tools/autograd/gen_variable_type.py

```diff
@@ -135,7 +135,8 @@
 std::shared_ptr<jit::tracer::TracingState> tracer_state;
 if (jit::tracer::isTracing()) {
   tracer_state = jit::tracer::getTracingState();
-  node = tracer_state->graph->create(jit::aten::${trace_name}, /*num_outputs=*/0);
+  const auto op_name = jit::Symbol::fromQualString("aten::${trace_name}");
```

Comment:
Making it static might or might not make things a bit fastee

Reply:
Yeah, I though I did write static, but I must have deleted it.